src/Billable.php

```diff
@@ -430,17 +430,28 @@ public function paymentMethods($parameters = [])
             return collect();
         }
 
-        $parameters = array_merge(['limit' => 24], $parameters);
+        $parameters = array_merge(['limit' => 24, 'customer' => $this->stripe_id], $parameters);
 
         // "type" is temporarily required by Stripe...
         $paymentMethods = StripePaymentMethod::all(
-            ['customer' => $this->stripe_id, 'type' => 'card'] + $parameters,
+            ['type' => 'card'] + $parameters,
```

Comment:
Doesn't the below work?

```suggestion
            ['type' => ['card', 'sepa_debit']] + $parameters,
```

Reply:
let me check once more .. IIRC I ran into issues.

Reply:
unfortunately this gives an `Invalid string: {:"0"=>"card", :"1"=>"sepa_debit"}`